lib/cocoapods/sandbox/file_accessor.rb

```diff
@@ -310,13 +310,24 @@ def docs
       def developer_files
         podspecs = specs
         result = [module_map, prefix_header]
+        spec_license = nil
+        if license_path = spec_consumer.license[:file]
+          license_path = root + license_path
+          if File.exists?(license_path)
+            spec_license = license_path
+          else
+            UI.warn "A license was specified in podspec `#{spec.name}` but the file does not exist - #{license_path}"
+          end
+        end
+
         if podspecs.size <= 1
-          result += [license, readme, podspecs, docs]
+          license_found = license
```

Comment:
any reason for this extra reference here?

Reply:
In this case, I want the spec license *or* the auto detected glob, but the pathname returned here might not exist if the Podspec specifies a path that doesn't exist. Couldn't think of a better way to keep that logic and also have the warning printed above

Reply:
@dnkoutso  Reworked this a bit, let me know what you think